test/Microsoft.SqlTools.ServiceLayer.UnitTests/ExecutionPlan/ExecutionPlanTests.cs

```diff
@@ -13,19 +13,19 @@
 using System.IO;
 using System.Reflection;
 using NUnit.Framework;
-using Microsoft.SqlTools.ServiceLayer.ShowPlan;
+using Microsoft.SqlTools.ServiceLayer.ExecutionPlan;
 
 
-namespace Microsoft.SqlTools.ServiceLayer.UnitTests.ShowPlan
+namespace Microsoft.SqlTools.ServiceLayer.UnitTests.ExecutionPlan
 {
-    public class ShowPlanXMLTests
+    public class ExecutionPlanXMLTests
```

Comment:
This looks to be testing a showplan.xml file specifically so I'd probably just leave this as ShowplanXMLTests

Same for the filenames of the test files themselves.